diff --git a/src/resources/lib/modules/services.py.old b/src/resources/lib/modules/services.py
index 6836a22..0cd3bda 100644
--- a/src/resources/lib/modules/services.py.old
+++ b/src/resources/lib/modules/services.py
@@ -173,6 +173,7 @@ class services:
                     'order': 2,
                     'name': 32201,
                     'not_supported': [],
+                    'hidden': 'true',   # disable in tbox!
                     'settings': {
                         'ssh_autostart': {
                             'order': 1,
@@ -368,6 +369,7 @@ class services:
                     self.struct['ssh']['settings']['ssh_autostart']['value'] = '1'
                     self.struct['ssh']['settings']['ssh_autostart']['hidden'] = 'true'
                 cmd_file.close()
+                self.struct['ssh']['hidden'] = 'true'   # disable in tbox
             else:
                 self.struct['ssh']['hidden'] = 'true'
 
@@ -468,7 +470,7 @@ class services:
                 options['SSHD_DISABLE_PW_AUTH'] = '"%s"' % val
             else:
                 state = 0
-            self.oe.set_service('sshd', options, state)
+            #self.oe.set_service('sshd', options, state)
             self.oe.set_busy(0)
             self.oe.dbg_log('services::initialize_ssh', 'exit_function', 0)
         except Exception, e:
@@ -629,6 +631,7 @@ class services:
             self.initialize_ssh()
             self.load_values()
             self.set_wizard_buttons()
+            self.oe.notify('ssh', 'ssh service it NOT supported')
             self.oe.dbg_log('services::wizard_set_ssh', 'exit_function', 0)
         except Exception, e:
             self.oe.dbg_log('services::wizard_set_ssh', 'ERROR: (%s)' % repr(e))
